python/pipeline/experiment.py

```diff
@@ -333,6 +333,73 @@ def plot_calibration_curve(self):
         return fig
 
 
+@schema
+class SurgeryType(dj.Lookup):
+    definition = """ # Surgery types used in experiment.Surgery
+    surgery_type                : varchar(64)         # Types of surgery performed on mice
+    """
+    contents = [
+        ['Cranial Window and Headbar'],
+        ['Headbar'],
+        ['Burr Hole and Suture'],
+        ['C-Section'],
+        ['Viral Injection'],
+        ['Electroporation']
+    ]
+
+
+@schema
+class SurgeryOutcome(dj.Lookup):
+    definition = """ # Surgery outcomes used in experiment.Surgery
+    surgery_outcome             : varchar(32)         # Possible outcomes of surgeries performed on mice
+    """
+    contents = [
+        ['Survival'],
+        ['Non-Survival'],
+    ]
+
+
+@schema
+class SurgeryQuality(dj.Lookup):
+    definition = """ # Surgery qualities used in experiment.Surgery
+    surgery_quality              : varchar(32)         # Possible qualities of surgeries performed on mice
```

Comment:
Could we better have something like a 1-5 or 1-100 ranking (that way we don't need this table, it will just be an int), it will still be qualitative but maybe finer grained.